android/customtabs.html

```diff
@@ -1,5 +1,10 @@
 {{+bindTo:partials.standard_multidevice_article}}
 
+<meta name="robots" content="noindex">
+<script>
+   window.location.replace('https://developers.google.com/web/android/custom-tabs/');
```

Comment:
Any reason not to use https://css-tricks.com/snippets/html/meta-refresh/ instead?

Reply:
Not particularly - I just saw other places in the code were doing like this and figured there must be a reason: https://github.com/GoogleChrome/multi-device/blob/master/android/installtohomescreen.html

Reply:
Okay, fair enough.